### ISSUE TITLE
Go modules are not published correctly

### ISSUE THREAD

**Issue by grantseltzer** [aquasecurity/tracee#1023]
I realized when upgrading the external package in tracee-rules that there's something wrong with how we publish our packages.

We have a [github.com/aquasecurity/tracee](https://pkg.go.dev/github.com/aquasecurity/tracee) package, which we stopped having after v0.4.0 but still is shown in pkg.go.dev

We have a [github.com/aquasecurity/tracee/tracee-ebpf](https://pkg.go.dev/github.com/aquasecurity/tracee/tracee-ebpf) package, which doesn't have corresponding tags.

The [github.com/aquasecurity/tracee/tracee-ebpf/external](github.com/aquasecurity/tracee/tracee-ebpf/external) package does not have corresponding tags. As a result people can't do something like `go get github.com/aquasecurity/tracee/tracee-ebpf/external@v0.6.1`.

**Comment by grantseltzer** [comment]
So i'm currently thinking that we should have one top level `github.com/aquasecurity/tracee` module. This would greatly simplify things, a downside being that we have to update usage of packages as we update the packages themselves.

I've tested a single module with my personal fork https://github.com/grantseltzer/tracee/tags with [this test consumer](https://github.com/grantseltzer/test-tracee-mod) and it appears to work how I envision.

When I did this, there's another small problem we run into where pkg.go.dev appears to take a couple hours to take effect:
 https://github.com/golang/go/issues/39007
